che-incubator/chectl
# feat: Add license utility to chectl to validate and add license header to codebase

### What does this PR do?
This PR add a new automation to chectl to check and add license headers to codebase. Also this PR fix all incorrect license headers.

Refference issue: https://github.com/eclipse/che/issues/19647
### Screenshot/screencast of this PR



### What issues does this PR fix or reference?



### How to test this PR?

Download current branch and execute:
For check release:
```bash
/bin/bash .ci/check-license.sh --check-license
```
To add new license header to a file which doesn't contain a license:
```bash
/bin/bash .ci/check-license.sh --add-license
```
### PR Checklist

[As the author of this Pull Request I made sure that:](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#pull-request-template-and-its-checklist)


### Reviewers

Reviewers, please comment how you tested the PR when approving it.
